Aztec/Classes/Public/Extensions/NSAttributedString+Lists.swift

```diff
@@ -6,6 +6,23 @@ import UIKit
 //
 extension NSAttributedString
 {
+    /// Given a collection of NSRange's, this method will filter all of those who contain a TextList, and
+    /// doesn't match the specified Style.
+    ///
+    /// - Parameters:
+    ///     - ranges: Ranges to be filtered
+    ///     - style: Style to be matched
+    ///
+    /// - Returns: A collection of NSRange's, guarranteed to either *NOT* contain TextLists, or contain TextLists
```

Comment:
Suggestion to increase clarity:

`Returns: A subset of the input ranges that don't contain TextLists matching the input style.`

The description applies for both possible scenarios and is slightly easier to read.